<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QWidget" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QPushButton" name="numButton1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="numButton2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="numButton3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="numButton4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="numButton5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="numButton6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="shortcut">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="numButton7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="shortcut">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="numButton8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="shortcut">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="numButton9">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="shortcut">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="numButton0">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="shortcut">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="textedit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>441</width>
     <height>131</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255)</string>
   </property>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>96</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="plusButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>390</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minusButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>390</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="timeButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
   <property name="shortcut">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="divideButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
   <property name="shortcut">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ACButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DELButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>DEL</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftPButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>35</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>(</string>
   </property>
   <property name="shortcut">
    <string>(</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightPButton">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>400</y>
     <width>35</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>)</string>
   </property>
   <property name="shortcut">
    <string>)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dotButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>35</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
   <property name="shortcut">
    <string>.</string>
   </property>
  </widget>
  <zorder>numButton1</zorder>
  <zorder>numButton2</zorder>
  <zorder>numButton3</zorder>
  <zorder>numButton4</zorder>
  <zorder>numButton5</zorder>
  <zorder>numButton6</zorder>
  <zorder>numButton7</zorder>
  <zorder>numButton8</zorder>
  <zorder>numButton9</zorder>
  <zorder>numButton0</zorder>
  <zorder>widget</zorder>
  <zorder>plusButton</zorder>
  <zorder>minusButton</zorder>
  <zorder>timeButton</zorder>
  <zorder>divideButton</zorder>
  <zorder>textedit</zorder>
  <zorder>ACButton</zorder>
  <zorder>DELButton</zorder>
  <zorder>leftPButton</zorder>
  <zorder>rightPButton</zorder>
  <zorder>dotButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
